.babelrc

```diff
@@ -4,27 +4,4 @@
     "flow"
   ],
   "retainLines": true,
-  "plugins": [
-    [
-      "module-resolver",
-      {
-        "root": [
```

Comment:
Is there a way to keep this? relative imports is a PITA in a big project like this.